frontend/pages/index.page.tsx

```diff
@@ -157,28 +164,10 @@ const AboutModularHistory: FC = () => {
   );
 };
 
-const FeaturedContent: FC = () => {
-  const [items, setItems] = useState<Exclude<ModuleUnion, Topic>[]>([]);
-  const [loading, setLoading] = useState(true);
-
-  useEffect(() => {
-    const cancelTokenSource = axios.CancelToken.source();
-    axiosWithoutAuth
-      .get("/api/home/features/", { cancelToken: cancelTokenSource.token })
-      .then((response) => {
-        setItems(response.data);
-        setLoading(false);
-      })
-      .catch((error) => {
-        if (axios.isCancel(error)) return;
-        console.error(error);
-        setLoading(false);
-      });
-    return () => {
-      cancelTokenSource.cancel("component unmounted");
-    };
-  }, [loading]);
-
+const FeaturedContent: FC<FeaturedContentProps> = ({
+  featuredcontentData,
+}: FeaturedContentProps) => {
+  const items = featuredcontentData || [];
```

Comment:
If we default to `[]` instead of `{}` in `getStaticProps`, we won't need this check here and we can just always reference `featuredcontentData` instead of `items`.